content/java/concepts/calendar/terms/getActualMinimum/getActualMinimum.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.getActualMinimum()'
+Description: 'Returns the minimum value that the specified calendar field could have based on the time value of this calendar'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Calendar'
+  - 'Time'
```

Comment:
```suggestion
  - 'Date'
  - 'Time'
```
Need to use only tags from this list: https://github.com/Codecademy/docs/blob/main/documentation/tags.md